docs/kb/semgrep-secrets/per-product-ignore-not-working.md

```diff
@@ -0,0 +1,11 @@
+---
+description: Product specific path ignores requires a supported CLI version
+tags: 
+    - Semgrep Secrets
+---
+
+# Why didn't Semgrep Secrets ignore the files/folders I specified?
```

Comment:
```suggestion
# Why didn't Semgrep Secrets ignore the files and folders I specified?
```

Reply:
ah! TIL, this is a great source, thanks for the link.